translations/story/04/1008/004 (その目、逸らさずに).json

```diff
@@ -0,0 +1,641 @@
+            "nextBlock": 51,
+            "pathId": 2730282357272224344,
+            "blockIdx": 50
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "（俺はっ……俺の思う『カッコいい』を、貫いてやるっ！！）",
+            "enText": "(I… I'm going to reach my ideal of \"cool\"!!)",
+            "nextBlock": 52,
+            "pathId": 569357729932559009,
+            "blockIdx": 51
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "っだぁああああああーーーー！！",
+            "enText": "Haaaaaaaaaaaa!",
+            "nextBlock": 53,
+            "pathId": 493801121672753552,
+            "blockIdx": 52
+        },
+        {
+            "jpName": "モノローグ",
+            "enName": "",
+            "jpText": "――勝負を制したのは、ナリタブライアンだった。",
+            "enText": "The winner of the bout ended up being Narita Brian.",
+            "nextBlock": 54,
+            "pathId": 5537897804440718017,
+            "blockIdx": 53
+        },
+        {
+            "jpName": "モノローグ",
+            "enName": "",
+            "jpText": "だがウオッカは最後の直線まで、\r\n見事“怪物”に喰らいつき続けたのだった……。",
+            "enText": "But even so, Vodka held on admirably against \nthe \"Monster\", right up until the final straight.",
+            "nextBlock": 55,
+            "pathId": 6106085115872578318,
+            "blockIdx": 54
+        },
+        {
+            "jpName": "ナリタブライアン",
+            "enName": "",
+            "jpText": "面白いヤツだな、オマエ。\r\n……また走ろう。",
+            "enText": "You're pretty amusing. \nLet's race again sometime.",
+            "nextBlock": 56,
+            "pathId": 5172454852435134840,
+            "blockIdx": 55
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "あっ……！　あ、ありがとーございましたっ、先輩！！",
+            "enText": "T-Thank you very much!!",
+            "nextBlock": 57,
+            "choices": [
+                {
+                    "jpText": "カッコよかったぞ、ウオッカ！！",
+                    "enText": "That was really cool, Vodka!",
+                    "nextBlock": 57
+                },
+                {
+                    "jpText": "カッコよかったよ、ウオッカ！！",
+                    "enText": "That was really cool, Vodka!",
+                    "nextBlock": 57
+                }
+            ],
+            "pathId": -7805213721475104835,
+            "blockIdx": 56
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "トレーナー……へへっ、ありがとな！！",
+            "enText": "Trainer… heh, really, thank you!!",
+            "nextBlock": 58,
+            "pathId": 781363766885449934,
+            "blockIdx": 57
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "……俺さ、もっともっとカッコよくなるから。\r\n見逃さねぇでくれよな！",
+            "enText": "…Y'know, I'm gonna become much more \ncooler from here on, so keep watching me!",
```

Comment:
More of an editing thing (sorry), but since I noticed: "much more cooler" sounds weird.

Reply:
Fair.